go/registry/api/api.go

```diff
@@ -483,20 +483,12 @@ func VerifyRegisterNodeArgs( // nolint: gocyclo
 		// by the entity signing key.
 		switch {
 		case sigNode.MultiSigned.IsSignedBy(entity.ID):
-			// Entity signing node registrations is feature-gated by a consensus
-			// parameter, and a per-entity configuration option.
-			//
-			// Ignore the per-entity option during sanity checks as the entity
-			// could have been modified (unless this is during genesis).
-			if !params.DebugAllowEntitySignedNodeRegistration || (!entity.AllowEntitySignedNodes && (!isSanityCheck || isGenesis)) {
-				logger.Error("RegisterNode: registration signed by entity",
-					"signed_node", sigNode,
-					"node", n,
-				)
-				return nil, nil, fmt.Errorf("%w: registration signed by entity", ErrInvalidArgument)
-			}
-
-			expectedSigners = append(expectedSigners, entity.ID)
+			// Entity-signed node registrations are not allowed.
```

Comment:
This should be changed to no longer check for entity signature at all. It should probably always use the second check.